ramls/usergroup.json

```diff
@@ -16,6 +16,10 @@
       "description": "A UUID identifying this group",
       "type": "string"
     },
+    "expirationOffsetInDays": {
+      "description": "The standard expiration offset in days of this group",
```

Comment:
What impact does this value have, e.g. do groups expire?

Reply:
This will be work of another story [UIU-1907](https://issues.folio.org/browse/UIU-1907).

In short, if expirationOffsetInDays is set for a user group, the expiration date field is populated by the current date plus the given offset in the frontend. If the selected patron does not have a defined offset, everything will work as it currently does.

Reply:
> the expiration date field is populated by the current date plus the given offset in the frontend. If the selected patron does not have a defined offset, everything will work as it currently does.

Is that the expiration date for the user (not the group)?

If so, could it be helpful for folks using this documentation to understand this property affects the expiration date for users within this group?

Reply:
> Is that the expiration date for the user (not the group)?

Yes, this will be expiration date for the user, based on the user's group. If the value is defined for the user's group, the frontend will pre-fill the corresponding field in the front-end based on this value. However, the person which is editing the user is able to change the pre-filled value.

> If so, could it be helpful for folks using this documentation to understand this property affects the expiration date for users within this group?

Yes, I may have to change the description to something like the following: "The standard expiration offset in days of a user within this group". Does that answer your question?
And I am not sure, if the description should specify that the value is used by the frontend?

Reply:
> The standard expiration offset in days of a user within this group

By standard offset do you mean the default period after which a newly created user will expire?

Reply:
Yes, this is the default period in days after which a newly created user will expire.

Reply:
Can you change the description to clarify the intended usage?
`The default period in days after which a newly created user that belongs to this group will expire.`
We should not mention how creates new users, this is implementation and institution dependent. Some might be created in the front-end, some might be created by some back-end batch process, some might be created by some external software (central campus registry).

Reply:
@julianladisch I updated the description as you suggested. It's a more meaningful description.

And you are right. New user may be created by other systems than the front-end.